Source/Krypton Toolkit/Examples/CircularProgressBar/Readme.md

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Add reference to this, via the start document

Reply:
Done